transcom/mymove
# B-17907-INT KICKBACK

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-17907)

## Summary

Initial worked failed to track vehicle description. Also PO requested PPM submission for closeout be tracked.

### How to test

1. Get a move to the Upload PPM Documents stage.
2. Fill out required information and submit.
3. Login as an office user and access the move, and its move history.
4. Confirm the move history tracks a vehicle description, and the submitted PPM.
![image](https://github.com/transcom/mymove/assets/147535220/2591124e-ff49-48e6-81f2-dcf6173954e4)
![image](https://github.com/transcom/mymove/assets/147535220/d1e1971d-6e62-40bd-986e-d4d4148aa593)
